templates/resourcesFilterButton.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { classes } from 'core/js/reactHelpers';
+
+export default function ResourcesFilterButton (props) {
+  const {
+    model,
+    onClick,
+    resources,
+    selected,
+    _filter
+  } = props;
+
+  const buttonText = model._filterButtons[_filter];
+  const ariaLabel = model._filterAria[`${_filter}Aria`];
+
+  function resourcesHasType(resources, type) {
+    const hasType = resources.some(_.matcher({ _type: type }));
```

Comment:
Minor: you can just return this rather than setting a variable

Reply:
Agreed and updated.